libs/client-api/src/http.rs

```diff
@@ -215,6 +215,26 @@ impl Client {
     self.token.read().subscribe()
   }
 
+  /// Sign in with magic link
+  #[instrument(level = "debug", skip_all, err)]
+  pub async fn sign_in_with_magic_link(
+    &self,
+    email: &str,
+    redirect_to: Option<String>,
```

Comment:
Document how to use the redirect_to? what is the default redirect_to?

Reply:
Done